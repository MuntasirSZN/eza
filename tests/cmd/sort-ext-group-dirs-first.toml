bin.name = "eza"
args = "tests/itest --sort=extension --group-directories-first"
