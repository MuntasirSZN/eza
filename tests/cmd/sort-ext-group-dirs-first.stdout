01.city  02.apple  abc  test  ab  abc.mp3  a.txt